code/modules/crafting/recipes.dm

```diff
@@ -337,6 +337,12 @@
 		/obj/item/weapon/restraints/handcuffs/cable = 1
 	)
 	category = CAT_MISC
+	
+/datum/crafting_recipe/toysword
+	name = "Toy Sword" 
+	reqs[] = list(/obj/item/weapon/light/bulb= 1, /obj/item/stack/cable_coil = 1, /obj/item/stack/sheet/plastic = 4)
```

Comment:
Yeah. You have to have the handle, and a bunch of telescoping tubes. But I could reduce it if you want me to.